src/contentScript.ts

```diff
@@ -54,7 +54,7 @@ window.addEventListener("error", function (e) {
   }
 });
 
-window.addEventListener("unhandledrejection", function (e) {
+window.addEventListener("unhandledrejection", function (e: any) {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/pixiebrix/pixiebrix-extension/pullRequest?prid=7629148)
```suggestion
window.addEventListener("unhandledrejection", function (e: unknown) {
```

Reply:
Interesting. I assume this was an automated comment. Kinda strange that it appears under your account.

Reply:
@fregante That is strange is shows up as being from me...

The docs make it seem like it should show up as being from Codacy: https://docs.codacy.com/repositories-configure/integrations/github-integration/#annotations. I'll reach out to their support team

Reply:
For suggested fixes, there's a limitation in GitHub: https://docs.codacy.com/repositories-configure/integrations/github-integration/#suggest-fixes

> Because of a limitation from GitHub, the author of the comments is the user that enabled the GitHub integration and not Codacy.

I think the annotations that don't have fixes will come from Codacy

Reply:
You could create a new dummy user that can be used for this sort of things. `@pixiebrix-bot`